arch/arm/boot/dts/bcm2711-rpi-cm4.dts

```diff
@@ -65,6 +65,18 @@
 		enable-active-high;
 		gpio = <&expgpio 6 GPIO_ACTIVE_HIGH>;
 	};
+
+	cam1_reg: cam1_reg {
```

Comment:
I'd prefer these patches to go after the "Downstream rpi- changes" comment below.